.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: Build VS Code extension
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [prereleased, released]
```

Comment:
Debatable whether it makes more sense to use this vs. `published`.

[Descriptions of event types](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release):

`released`:
> A release was published, or a pre-release was changed to a release.

`prereleased`:
> A release was created and identified as a pre-release. A pre-release is a release that is not ready for production and may be unstable.

`published`:
> A release, pre-release, or draft of a release was published.

If you use `published` then it'll upload the artifacts when you save a draft release, which can be nice because then the artifacts are ready to go when notifications go out that the a new release has been published.  Or, if it fails, you get to deal with that _before_ the release is finalized, potentially avoiding the need for a re-tag or patch release.